### PR TITLE
Add sed and coreutils when make is present

### PR DESCRIPTION
**Pull request by jdblischak**
I tested with the package r-fs.

cc: @isuruf
xref: #19, https://github.com/conda/conda-build/pull/2963

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -95,8 +95,12 @@
...
 
             # Skip build on win32
-            line = re.sub('  number: 0',
-                          '  number: 0\n  skip: true  # [win32]', line)
+            if line == '  number: 0\n':
+                line = '  number: 0\n  skip: true  # [win32]\n'
+
+            # Add sed and coreutils when make is present
+            if line == '    - {{posix}}make\n':
+                line = '    - {{posix}}make\n    - {{posix}}sed\n    - {{posix}}coreutils\n'
```
Comment (isuruf) [inline run.py]: Needs `# [win]` selector for both.

Comment (jdblischak) [reply run.py]: Thanks @isuruf. I missed the `sel_src_and_win` in your PR.

I added two spaces before the selector. Alternatively I could vertically align the selector with the others. This is what is currently creates:

```
  build:
    - {{ compiler('c') }}        # [not win]
    - {{ compiler('cxx') }}      # [not win]
    - {{native}}toolchain        # [win]
    - {{posix}}filesystem        # [win]
    - {{posix}}sed               # [win]
    - {{posix}}grep              # [win]
    - {{posix}}autoconf
    - {{posix}}automake-wrapper  # [win]
    - {{posix}}automake          # [win]
    - {{posix}}pkg-config
    - {{posix}}make
    - {{posix}}sed  # [win]
    - {{posix}}coreutils  # [win]
```

**Comment by bgruening** [comment]
Puh, thanks for tracking this down @isuruf and @jdblischak!

**Comment by isuruf** [comment]
This is done by conda-build now

**Comment by jdblischak** [comment]
@isuruf Thanks for the update. Please see PR #26

**Pull request merged**